docs/contributing/test.md

```diff
@@ -148,7 +148,7 @@ You can use the `TESTDIRS` environment variable to run unit tests for
 a single package.
 
 ```bash
-$ TESTDIRS='opts' make test-unit
+$ TESTDIRS='github.com/docker/docker/opts' TESTFLAGS='-test.run $^TestValidateIPAddress$' make test-unit
```

Comment:
This section talks about the `TESTDIRS` environment variable, so I guess this should be;

```suggestion
$ TESTDIRS='github.com/docker/docker/opts' make test-unit
```